cluster-scope/overlays/nerc-ocp-test-2/nodenetworkconfigurationpolicies/vlan-211.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: vlan-211
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: "wrk-5"
+  desiredState:
+    interfaces:
+    - name: eno1np0.211
+      type: vlan
+      state: up
+      ipv4:
+        enabled: true
+        dhcp: true
+        auto-routes: true
+        address:
+        - ip: 10.0.120.39
```

Comment:
Why are we setting a static ip here if we have `dhcp: true`?

Reply:
This should be set to false will make the change